robottelo/ui/factory.py

```diff
@@ -124,11 +124,11 @@ def make_org(session, force_context=False, **kwargs):
         u'select': True,
     }
     page = session.nav.go_to_org
-    core_factory(create_args, kwargs, session, page, force_context=False)
+    core_factory(create_args, kwargs, session, page)
     Org(session.browser).create(**create_args)
 
 
-def make_loc(session, force_context=False, **kwargs):
+def make_loc(session, force_context=True, **kwargs):
```

Comment:
1) `force_context` is used to reset org and loc even if they are already selected. 
2) Realised that there could be no usecase where in we would require org and loc to be selected for creating org or location itself again.
3) Hence it's removal, may be we can add it back if some usecase pops up. :)